src/main/java/javaslang/collection/IndexedSeq.java

```diff
@@ -106,7 +106,14 @@ default T last() {
             return get(length() - 1);
         }
     }
-
+    
+    @Override
+    default int lastIndexWhere(Predicate<? super T> p, int end) {
+        int i = Math.min(end, length()-1);
+        while(i >= 0 && !p.test(this.get(i))) i--;
```

Comment:
This is a good example where no curly braces are needed! But in most other cases we write them...